vhdbuilder/packer/components.json

```diff
@@ -355,12 +355,10 @@
     {
       "downloadURL": "mcr.microsoft.com/oss/kubernetes/ingress/nginx-ingress-controller:*",
       "amd64OnlyVersions": [
-        "0.19.0",
```

Comment:
I think you can remove them all

Reply:
I see a bunch of them being referenced in RP(_images.tpl) 
{- else if eq .component "http-application-routing-nginx-ingress-controller" -}}
  {{- if semverCompare ">=1.22.0" .version -}}1.2.1
  {{- else if semverCompare ">=1.21.0" .version -}}0.49.3
  {{- else -}}0.19.0
  {{- end -}}